src/main/java/seedu/duke/common/Constants.java

```diff
@@ -70,9 +71,23 @@
 
     //Command
     public static final String HEAD = "HEAD";
-    
+
     //Logging
     public static final String LOGGER_PATH = "GULIOLog.log";
     public static final String LOGGER_NAME = "LOGGER";
 
+    //Editor
+    public static final int TEXT_EDITOR_WIDTH = 800;
+    public static final int TEXT_EDITOR_HEIGHT = 800;
+    public static final int TEXT_AREA_WIDTH = 700;
+    public static final int TEXT_AREA_HEIGHT = 700;
+    public static final String DEFAULT_FONT_STYLE = "Times New Roman";
+    public static final int DEFAULT_FONT_SIZE = 16;
+    public static final int BOX_WIDTH = 50;
+    public static final int BOX_HEIGHT = 25;
+    public static final String FONT_SIZE_ICON = "Font size: ";
+    public static final String FONT_COLOUR_ICON = "Font colour";
+    public static final String SAVE_ICON = "Save";
```

Comment:
Is there a reason why the word "icon" is used here instead of something like "string"?